examples/01-filter/surface_reconstruction.py

```diff
@@ -0,0 +1,30 @@
+"""
+.. _surface_reconstruction_example:
+
+Surface Reconstruction
+~~~~~~~~~~~~~~~~~~~~~~
+
+Surface reconstruction has a dedicated filter in PyVista and is
+handled by :func:`pyvista.DataSetFilters.reconstruct_surface`.  This
+tends to perform much better than :func:`DataSetFilters.delaunay_3d`.
```

Comment:
Should we link to this new filter from `DataSetFilters.delaunay_3d`?

Reply:
Yes, that's a good point.